### PR TITLE
Fix/issue with chrome version

### PR DESCRIPTION
**Pull request by MDaams**
I had to modify the Nightwatch config in the project template to make Nightwatch work with the latest chrome version (see https://github.com/nightwatchjs/nightwatch/releases/tag/v1.1.12).

### CODE REVIEW

**Comment by pascalw** [comment]
Nice work! 👍

File: project_template/package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "babel-eslint": "^7.2.3",
     "babel-loader": "^8.0.0",
     "chai": "^3.5.0",
-    "chromedriver": "^2.35.0",
+    "chromedriver": "^75.0.0",
```
Comment (pascalw) [inline project_template/package.json]: Latest version is `76.0.0`. Might as well use that?

Comment (MDaams) [reply project_template/package.json]: Agree.

Comment (MDaams) [reply project_template/package.json]: Changed it.

**Pull request merged**